src/main/kotlin/calculator/StringSeparator.kt

```diff
@@ -0,0 +1,24 @@
+package calculator
+
+object StringSeparator {
+
+    private const val DEFAULT_DELIMITER = ",|:"
+    private val CUSTOM_REGEX = Regex("//(.)\n(.*)")
+    private val DEFAULT_NUMBER = PositiveNumbers(listOf(PositiveNumber(0)))
+
+    fun separate(inputString: String?): PositiveNumbers {
+        if (inputString.isNullOrEmpty()) {
+            return DEFAULT_NUMBER
+        }
+
+        return CUSTOM_REGEX.find(inputString)
+            ?.let { toPositiveNumbers(inputString = it.groupValues[2], delimiter = it.groupValues[1]) }
+            ?: toPositiveNumbers(inputString)
+    }
+
+    private fun toPositiveNumbers(inputString: String, delimiter: String = DEFAULT_DELIMITER): PositiveNumbers = inputString.split(delimiter.toRegex())
```

Comment:
```suggestion
    private fun toPositiveNumbers(
        inputString: String,
        delimiter: String = DEFAULT_DELIMITER
    ): PositiveNumbers = inputString.split(delimiter.toRegex())
```
가로로 너무 길어지는 경우에는 가독성이 떨어져요.
이럴 때에는 개행을 하면 좀 더 코드 읽기가 편해져요.